pelican/__init__.py

```diff
@@ -538,9 +540,9 @@ def main(argv=None):
     except KeyboardInterrupt:
         logger.warning('Keyboard interrupt received. Exiting.')
     except Exception as e:
-        logger.critical('%s', e)
+        logger.critical("%s: %s" % (e.__class__.__name__, e))
 
         if args.verbosity == logging.DEBUG:
-            raise
+            console.print_exception()
```

Comment:
This makes process finish "successfully" (return code 0). Should be non-zero. Probably easier to remove `else` and run that block unconditionally:
```python
        if args.verbosity == logging.DEBUG:
            console.print_exception()

        sys.exit(getattr(e, 'exitcode', 1))
```

Reply:
done!